sdk/core/core-amqp/src/retry.ts

```diff
@@ -54,6 +53,38 @@ export enum RetryOperationType {
   session = "session"
 }
 
+/**
+ * Retry policy options for operations on the EventHubClient.
```

Comment:
Since this is no longer just for EventHubClient, the comment needs to change

```suggestion
 * Retry policy options that determine the mode, number of retries, retry interval etc.
```